tests/export_cache/test_export_cache.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""
+Test basic usage of the mock code on examples using aiida-diff.
+"""
+
+import os
+import pytest
+
+from aiida.engine import run_get_node
+from aiida.plugins import CalculationFactory, DataFactory
+from aiida.engine import WorkChain
+from aiida.engine import ToContext
+from aiida.orm import Node
+from aiida.orm.querybuilder import QueryBuilder
+
+DiffParameters = DataFactory('diff')
+DiffCalculation = CalculationFactory('diff')
+CALC_ENTRY_POINT = 'diff'
+
+#### diff workchain for basic tests
+
+
+class DiffWorkChain(WorkChain):
```

Comment:
Add a ``# type: ignore`` comment here. The reason for this ``mypy`` issue is that ``aiida-core`` does not have type annotations, so it gets confused when you try to subclass from ``WorkChain``.